src/providers/OrganizationProvider.tsx

```diff
@@ -104,7 +105,7 @@ export default function OrganizationProvider({ children }: OrganizationProviderP
   }, [reloadOrgPreferences, selectedOrganization]);
 
   useEffect(() => {
-    if (organizations.length && userPreferences) {
+    if (userBootstrapped && organizations.length && userPreferences) {
```

Comment:
select the org only after user is bootstrapped - so we have the preferences data. this was the race condition - preferences being available before getting to this code.. which doesn't always happen, esp in prod